extension/src/standalone_pages/InstructorDashboardRenderVideoInfo.js

```diff
@@ -138,7 +138,32 @@ function renderActiveFeedback(video, response) {
     };
 }
 
-function renderPassiveFeedback(video, response) {
+function displayContextKeyWords(label, keywordsMap) {
+    
+    if (!keywordsMap[label]) {
+        Swal.fire({
+            title: `Keywords at timestamp ${label}`,
+            html: 
+                `<html>
+                    <div class="spinner-border text-success" role="status">
+                        <span class="sr-only">Loading...</span>
+                    </div>
+                </html>`,
+            confirmButtonText: "Close"
+        });
+    } else {
+        Swal.fire({
+            title: `Keywords at timestamp ${label}`,
+            html: 
+                `<html>
+                    ${keywordsMap[label]}
```

Comment:
For the first iteration of this pull request, this looks fine. See my comments below on the presentation